### DISCUSSION TITLE
Problem with executing script "FirmwareImage.ps1"

### DISCUSSION THREAD

**Discussion by tomekjen** [PeterPawn/YourFritz#59] [General]
Hello,
When I try to create image for 7530 with script "FirmwareImage.ps1" I got error "Method invocation failed because [System.String] does not contain a method named 'getBootableImage' ".
Can anyone tell me why I did that error, or what to do with it ?
I tried on Windows 7 (with upgraded Powershell) and on Windows 10. 

<img width="910" alt="Zrzut ekranu 2023-08-12 o 21 38 24" src="https://github.com/PeterPawn/YourFritz/assets/18482263/87ea516f-4727-4513-b76a-39b0bfc63762">

**Comment by PeterPawn** [comment]
You've to source the script itself first - instead of the first `>>` a newline (aka ENTER key) has to be used.

**Reply by tomekjen** [reply]
Thank you for responce - now it is clear to me how to process :)
